### ISSUE TITLE
Import-Module HPEOneview.630 - Drive with the name 'HKLM' does not exist

### ISSUE THREAD

**Issue by Synovialbasher4** [HewlettPackard/POSH-HPOneView#680]
Hello, 

I'm having an error when importing the HPEOneview.630 module. The error seems to be related to a Windows registry, however this is on a Linux system. 

**System Configuration**
Oracle Linux 9.1
Powershell version 7.4.5
HPE Oneview module 6.30

**Expected Action**
HPEOneview.630 module is loaded successfully 

**Issue**
When attempting to import the HPEOneview.630 module, an error occurs: 

`PS /home/user> Import-Module HPEOneview.630

Get-Item: Cannot find drive. A drive with the name 'HKLM' does not exist.
Import-Module: The library is unable to load due to this sytem missing the required .NetFramework 4.7.2 client.  Please visit https://go.microsoft.com/fwlink/?LinkId=863265 to download the .NetFramework 4.7.2 Offline Installer.`

**Comment by ChrisLynchHPE** [comment]
The OneView 6.30 library is not supported anymore.  Is there a reason preventing you from updating the library?  Is your appliance also at 6.30?

**Comment by ChrisLynchHPE** [comment]
Additionally, I just installed the most currently published 6.30 library on an Ubuntu WSL image (22.04 with PowerShell 7.4.6), and the module loaded just fine:

```powershell
PS /home/chris> ipmo hpeoneview.630
PS /home/chris> $error
New-Item: Access to the path '/usr/local/share/powershell' is denied.

New-Item: Access to the path '/usr/local/share/powershell' is denied.

PS /home/chris> get-module HPEOneView.630

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     6.30.3188…            HPEOneView.630                      {Add-OVApplianceTrustedCertificate, Add-OVB…
```

**Comment by Synovialbasher4** [comment]
Thank you, we see that it is working with the latest version of Oneview (7.4.6) that was just released.
